internal/pkg/config/v1/config.go

```diff
@@ -133,6 +133,24 @@ func New() *Config {
 	}
 }
 
+func (c *Config) DecryptContextStates() error {
+	for _, context := range c.Contexts {
```

Comment:
Don't we only want to decrypt the tokens for the current state? There should be no need to decrypt tokens for a state we're not using.

Reply:
you're right. Changed to only decrypt for current context!